plugins/sfThemePlugin/config/sfThemePluginConfiguration.class.php

```diff
@@ -1,21 +0,0 @@
-<?php
```

Comment:
I've updated the PR description, I'll use it as the commit message.

Thanks @MikeFE